Bio/Seq.py

```diff
@@ -2006,7 +2006,7 @@ def __init__(self, data, length=None):
                     if start < end:
                         raise ValueError("Sequence data are overlapping.")
                     elif start == end:
-                        _data[current] += seq
+                        _data[current] += seq  # noqa
```

Comment:
Why use a ``# noqa``, and specifically why a blanket one rather than listing a single flkae8 code?

Reply:
Is this to do with the curious choice of ``_data`` with a leading underscore as a local variable? Could it not be ``tmp_data`` or ``local_data`` instead?

Reply:
> Why use a # noqa, and specifically why a blanket one rather than listing a single flkae8 code?

Because otherwise flake8 seems to think that `current` is an undefined variable.